### ISSUE TITLE
[css-fonts] `CSSFontFaceDescriptors` is missing descriptor attributes

### ISSUE THREAD

**Issue by cdoublev** [w3c/csswg-drafts#10107]
[`CSSFontFaceDescriptors`](https://drafts.csswg.org/css-fonts-4/#cssfontfacedescriptors) seems to be missing `font-weight` (not `fontWeight`) and [`font-stretch`/`fontStretch`](https://drafts.csswg.org/css-fonts-4/#font-face-font-stretch) (legacy).

CSS Fonts 5 may also missing a partial interface definition with [`font-size`/`fontSize`](https://drafts.csswg.org/css-fonts-5/#descdef-font-face-font-size), [`size-adjust`/`sizeAdjust`](https://drafts.csswg.org/css-fonts-5/#size-adjust-desc), and `*-override`/`*Override`.

A simpler alternative is proposed in #10105.

**Comment by svgeesus** [comment]
Agreed, the missing `font-weight` looks like a simple error and the legacy values are an over-eager rename rather than a [deprecation, which was the csswg resolution](https://github.com/w3c/csswg-drafts/issues/551#issuecomment-1885196273).

I don't follow about the "simpler alternative".

**Comment by cdoublev** [comment]
`CSSFontFaceDescriptors` attributes would be defined generically, like [`CSSStyleProperties` attributes](https://drafts.csswg.org/cssom-1/#dom-cssstyledeclaration-camel-cased-attribute).

  > For each CSS descriptor `descriptor` that is a supported CSS descriptor accepted in `@font-face`, the following partial interface applies where `_camel_cased_attribute` is obtained by running the [CSS property to IDL attribute](https://drafts.csswg.org/cssom-1/#css-property-to-idl-attribute) algorithm for `descriptor`.
  >
  > ```
  > partial interface CSSFontFaceDescriptors {
  >   attribute [LegacyNullToEmptyString] CSSOMString _camel_cased_attribute;
  > };
  > ```

And similarly for dashed attributes.

**Comment by svgeesus** [comment]
Fixed in CSS Fonts 5 by 

 - https://github.com/w3c/csswg-drafts/commit/cf45c851665cfe6d55b04408d44ab14a5b5ef6ee 
 - https://github.com/w3c/csswg-drafts/commit/cd59f7683fd5533716e604aed08e4ff34f1dbf80
